### PR TITLE
IN WORLD

### PR DESCRIPTION
**Pull request by clusterfack**
Removes a bunch of in worlds from the code.
Deity please stop adding more it wounds me.

### CODE REVIEW

**Comment by DeityLink** [comment]
I'm only responsible for 
>for(var/turf/unsimulated/wall/bomberman/T in world)

from debug.dm, which you replaced with turfs, which didn't exist back then...

**Comment by clusterfack** [comment]
Pomf already fixed your mass drivers/pod doors/etc in world because they almost killed the server once. Also the remove all bomberman shit calls in world like 6 times.

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -131,7 +131,6 @@ var/list/department_radio_keys = list(
 		return
 
 	var/message_mode = get_message_mode(message)
-	var/message_mode_name = message_mode_to_name(message_mode)
```
Comment (d3athrow) [inline code/modules/mob/living/say.dm]: broke the build

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -160,7 +159,6 @@ var/list/department_radio_keys = list(
 		speaking = parse_language(message)
 		say_testing(src, "Getting speaking language, [istype(speaking) ? "got [speaking.name]" : "got null"]")
 	if(istype(speaking))
-		var/oldmsg = message
```
Comment (d3athrow) [inline code/modules/mob/living/say.dm]: broke the build

**Comment by clusterfack** [comment]
Oh I didn't notice I modified those

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 3]
```diff
@@ -97,7 +97,7 @@ rcd light flash thingy on matter drain
 
 	power_type = /mob/living/silicon/ai/proc/overload_machine
 
-/mob/living/silicon/ai/proc/overload_machine(obj/machinery/M as obj in world)
+/mob/living/silicon/ai/proc/overload_machine(obj/machinery/M in machines)
```
Comment (clusterfack) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: whoops I should have left as obj, that could cause problems for right clicking

**Pull request merged**